src/data/latest-updates/2025-04-11_increasing-data-security-improves-usability/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+path: /latest-updates/increasing-data-security
+date: 2025-04-10
```

Comment:
i didn't catch this before. is there a reason we have 4/10 here and 4/11 in the filename?

Reply:
thank you! it's been fixed now!